web/frontend/package.json

```diff
@@ -80,7 +80,7 @@
     "svelte": "^4.1.2",
     "svelte-check": "^3.5.2",
     "tailwindcss": "3.3.3",
-    "three": "0.159.0",
```

Comment:
Why not 167?

Reply:
I was just trying to match what we are using in prime/blocks, which uses 166.1, for dependency deduping. I can do 167 and also bump prime/blocks.

Reply:
Makes sense, no worries whatever unblocks us!